bin/cucumber-api

```diff
@@ -7,4 +7,6 @@ else
   path=cucumber/api/features/"$path"
 fi
 
+# TODO: cleanup overlap between between -r requires here and in yaml
+# TODO: add note about why we're using exec here
```

Comment:
Wouldn't this be a micro optimization?  This is something we ordinarily don't do in bash scripts, it seems whatever minor benefit isn't worth the possible confusion.  If you think it is, I can add a comment explaining what you did above.

Reply:
Depends - there is a fundamental difference in how signals are passed between parent and exec vs non-exec spawned process. If you use `exec` there is no ambiguity and all signals will work while non-exec may have issues. I don't know if that is why this line has it tbh so it would be a bit risky to remove I think.

Reply:
Fair enough I'll leave per Chesterton's fence.